entry/src/main/java/com/thoughtbot/expandablerecyclerview/entities/CommonHelper.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (C) 2020-21 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtbot.expandablerecyclerview.entities;
+
+import ohos.agp.colors.RgbColor;
+import ohos.agp.components.ComponentContainer;
+import ohos.agp.components.Image;
+import ohos.agp.components.element.ShapeElement;
+import ohos.agp.utils.Color;
+import ohos.agp.utils.TextAlignment;
+import ohos.agp.window.dialog.ToastDialog;
+import ohos.app.Context;
+import com.thoughtbot.expandablerecyclerview.ExpandableListAdapter;
+import com.thoughtbot.expandablerecyclerview.ExpandableListContainer;
+import com.thoughtbot.expandablerecyclerview.ResourceTable;
+import com.thoughtbot.expandablerecyclerview.util.ResUtil;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Parent helper class for each Ability Helper.
+ */
+public abstract class CommonHelper {
+    protected ExpandableListContainer mGroupContainer;
+    protected ArrayList<ParentChild> mGroupNameItem = new ArrayList<>();
+    protected ArrayList<String> mTempGroupNameItem = new ArrayList<>();
+    protected ArrayList<String> mTempChildNameItem = new ArrayList<>();
+    protected ArrayList<String> mFinalGroupNameItem = new ArrayList<>();
+    protected ArrayList<Integer> mGroupImageItem = new ArrayList<>();
+    protected ArrayList<String> mFavouriteItem = new ArrayList<>();
+    protected Context context;
+    protected ComponentContainer rootView;
+
+    /**
+     * This method is used to prepare group items.
+     */
+    public void getGroupItems() {
```

Comment:
these access specifier of this method can be modified to protected as it need not be used outside child classes..Please check the applicability for all the other methods in this class as well